ios/ElectronCash/electroncash_gui/ios_native/send.py

```diff
@@ -260,7 +270,7 @@ def viewWillAppear_(self, animated : bool) -> None:
             try:
                 qpt = list(self.queuedPayTo)
                 self.queuedPayTo = None
-                self.onPayTo_message_amount_(qpt[0],qpt[1],qpt[2])
+                self.onPayTo_message_amount_(qpt[0],qpt[1],qpt[2], None, None)
```

Comment:
```suggestion
                self.onPayTo_message_amount_(qpt[0],qpt[1],qpt[2])
```

Reply:
Given the signature change I left this code as
```python
self.onPayTo_message_amount_opReturn_isRaw_(qpt[0],qpt[1],qpt[2], None, False)
```